lib/mayaUsd/utils/util.h

```diff
@@ -152,6 +152,11 @@ double ConvertMDistanceUnitToUsdGeomLinearUnit(const MDistance::Unit mdistanceUn
 MAYAUSD_CORE_PUBLIC
 MDistance::Unit ConvertUsdGeomLinearUnitToMDistanceUnit(const double linearUnit);
 
+/// Convert the given \p mdistanceYnit into its text representation suitable
```

Comment:
small typo

Reply:
Will be fixed in next PR about units and up-axis in USD stages.